src/shared/src/components/MainContentNav.tsx

```diff
@@ -0,0 +1,65 @@
+import { makeStyles } from "@material-ui/core";
+import React from "react";
+import cn from 'classnames';
+
+export interface IMainContentNavProps {
+  navigateText: string;
+}
+
+const useStyles = makeStyles({
+  button: {
+    textDecoration: 'underline',
+    textAlign: 'left',
+    width: 'fit-content',
+  },
+  jumpNav: {
+    backgroundColor: "#fff",
+  },
+  'sr-only': {
+    position: 'absolute',
+    width: '1px',
+    height: '1px',
+    padding: 0,
+    margin: '-1px',
+    overflow: 'hidden',
+    clip: 'rect(0,0,0,0)',
+    border: 0,
+  },
+  'sr-only-focusable': {
+    '&:focus': {
+      position: 'relative',
+      width: 'auto',
+      height: 'auto',
+      clip: 'initial',
+      margin: 'inherit',
+      padding: 'inherit',
+      border: 'initial',
+    },
+  }
+});
+
+export function MainContentNav({
+  navigateText,
+}: IMainContentNavProps) {
+  const classes = useStyles();
+
+  const handleOnClick = () => {
+    // workaround because we still use a hash-router (sigh...)
+    // can be replaced by the more elegant solution <a href="#main-content></a> once this is no longer the case.
```

Comment:
AH, sorry I did not recall this, so disregard the comment about using links instead of buttons.
Maybe we can still use links, but change a query variable, so something like
https://.......#application-name?landmark=main-content
That way it might be possible to send a link to the landmark inside an app...
It's not really the way to go long term, as queries are usually before the hash and usually causes a page refresh when they change, but on the right hand they don't. Just taught maybe it could help us keep the buttons out.

Reply:
I would prefer to keep this as simple as possible for now. At some point we have to move away from hash-router, then this will all be solved by simple anchors to the relevant element id.